lib/data_handlers/json_handler.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+#  class for loading json data
+class JsonHandler
```

Comment:
I would have made this class a `Repository` e.g. `PersonJsonRepository` with a method for `load` rather than `get_person_dtos`.